packages/mdc-drawer/README.md

```diff
@@ -160,6 +160,62 @@ Drawers can contain a header element which will not scroll with the rest of the
 </aside>
 ```
 
+### Drawer Below Top App Bar
```

Comment:
Can we remove this section and keep the one in `Usage with Top App Bar` section?

Reply:
This is the permanent below top app bar...it is slightly different. Do you want to move it to the `Usage with Top App Bar` section?

Reply:
Let's keep just one example for Dismissible drawer variant to avoid maintaining two different variations of this version.